src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -590,6 +591,23 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
                 BindingInfo bindingInfo = binding.ToBindingInfo();
 
                 request.Metadata.Bindings.Add(binding.Name, bindingInfo);
+
+                binding.Properties.TryGetValue(ScriptConstants.SupportsDeferredBindingKey, out bool supportsDeferredBindingFlag);
+
+                if (supportsDeferredBindingFlag)
+                {
+                    binding.Properties.TryGetValue(ScriptConstants.SkipDeferredBindingKey, out bool skipDeferredBindingFlag);
+
+                    if (skipDeferredBindingFlag)
+                    {
+                        _workerChannelLogger.LogInformation("Function '{functionName}' skipping binding to ParameterBindingData for binding name '{name}' and type '{type}' as '{SkipDeferredBinding}' set to '{value}'", metadata.Name, binding.Name, binding.Type, ScriptConstants.SkipDeferredBindingKey, skipDeferredBindingFlag);
```

Comment:
@mathewc I have moved telemetry to Load request instead of per invocation. Wanted to double check if we need these log statements or only metric is sufficient. I have added log in case we will need information such as Binding type, name, but totally fine to remove them if not needed.